elyra/pipeline/processor_local.py

```diff
@@ -112,17 +129,68 @@ def _sort_operation_dependencies(operations_by_id: dict, ordered_operations: lis
                                                                         parent_operation)
             ordered_operations.append(operation)
 
-    def _get_envs(self, operation: Operation) -> Dict:
-        """Operation stores environment variables in a list of name=value pairs, while
-           subprocess.run() requires a dictionary - so we must convert.  If no envs are
-           configured on the Operation, the existing env is returned, otherwise envs
-           configured on the Operation are overlayed on the existing env.
-        """
-        envs = os.environ.copy()
-        for nv in operation.env_vars:
-            nv_pair = nv.split("=")
-            if len(nv_pair) == 2:
-                envs[nv_pair[0]] = nv_pair[1]
-            else:
-                self.log.warning(f"Could not process environment variable entry `{nv}`, skipping...")
-        return envs
+
+class OperationProcessor(ABC):
+
+    def __init__(self):
+        self.log = log.get_logger()
+
+    @abstractmethod
+    def operation_name(self) -> str:
+        pass
```

Comment:
This class should just return the member value and let the leaf classes set `self._operation_name` to the appropriate value and remove their property method since they will inherit this one.   This should also not be decorated with `@abstractmethod` since its just a vanilla base class method.
```suggestion
        return self._operation_name
```